Update section Internal Services (#255)
EGI-Federation/documentation: content/en/internal/configuration-database/access/_index.md

---
title: "Access"

> description: "Accessing the configuration database"

Reviewer: ```suggestion
description: "Accessing the Configuration Database"
```

I would capitalise the service name everywhere (so Configuration Database)

Author: I agree, in principle. But not everywhere the text is referring to the official service (with its official name). The term "configuration database" also makes sense on its own, when describing what it actually is or when discussing it. But in this case, yes, you are right.

weight: 25
type: "docs"
---

> To access [web interface](https://goc.egi.eu) of the EGI configuration database

Reviewer: ```suggestion
To access the [web interface](https://goc.egi.eu) of the EGI Configuration Database
```

Author: Fixed

> To access [web interface](https://goc.egi.eu) of the EGI Configuration Database

Reviewer: ```suggestion
To access the [web interface](https://goc.egi.eu) of the EGI Configuration Database
```

Reviewer: It seems this point is one of or even the latest one to be addressed before being able to merge, no?

(GOCDB), users can either:

- [Use EGI Check-in](#using-institutional-account-via-egi-check-in) to access

>   the GOCDB with an institutional account, or

Reviewer: ```suggestion
  the Configuration Database with an institutional account, or
```

We should favour the branded service name (Configuration Database) over the legacy/tool name (GOCDB).

Author: Fixed

- [Use an X.509 digital certificate](#using-an-x509-digital-certificate)
  installed in the browser.

Users can access the system as soon as they are authenticated. However,
they will only be able to update information based on their roles.
More information about roles and associated permission is available in the
[_Users and roles_](https://wiki.egi.eu/wiki/GOCDB/Input_System_User_Documentation#Users_and_roles)
section of the documentation.

Applications requesting a specific role have to be validated by parent roles
or administrators. Once granted, users can access and/or modify relevant

> information, according to the roles granted to the user.

Reviewer: ```suggestion
information, according to the roles granted to them.
```

## Using institutional account via EGI Check-in

In order to be able to access the GOCDB with their institutional account, users
need to:

1. Have their Identity Provider (IdP) federated in EGI Check-in (via
   [eduGAIN](https://edugain.org/) or directly)

> 2. Have created an [EGI Check-in

Reviewer 2: better leave md to do the numbering for you

Author: except in this case it does not do it correctly. my understanding is that there are 2 steps with a compliant idp, and 2 more steps, in case the idp is non-compliant. if there would not be the huge red note before step 3, the numbering would have been correct. are you sure you still want 2 sets of steps 1,2 then 1,2?

>    account](https://wiki.egi.eu/wiki/AAI_usage_guide)

Reviewer: This should probably point to a page in docs.egi.eu.

Reviewer: This comment from the previous review (and a few others) may have been hidden due to the collapse of message by GitHub.

Author: I will update this in another revision, the one about check-in. Let's leave it as it is for now, this is a sensible subject.

Reviewer: I do not really agree, https://wiki.egi.eu/wiki/AAI_usage_guide is already just a redirection https://docs.egi.eu/users/check-in/, let's be nicer with users and bring them directly to https://docs.egi.eu/users/check-in/ :)

Author: Fixed

reading the #Users and roles section of this documentation.
Applications requesting a specific role have to be validated by parent roles
or administrators. Once granted, users can access and/or modify relevant
information, according to the roles granted to the user.

## Using institutional account via EGI Check-in

In order to be able to access the GOCDB with their institutional account, users
need to:

1. Have their Identity Provider (IdP) federated in EGI Check-in (via
   [eduGAIN](https://edugain.org/) or directly) ;
   [eduGAIN](https://edugain.org/) or directly)
1. Have created an [EGI Check-in
   account](https://wiki.egi.eu/wiki/AAI_usage_guide) ;
   account](https://wiki.egi.eu/wiki/AAI_usage_guide)

{{% alert title="Important" color="warning" %}}
In the case the user cannot use an IdP compliant with [REFEDS
R&S](https://refeds.org/research-and-scholarship) and [REFEDS
Sirtfi](https://refeds.org/sirtfi), the user will have to request joining a
specific group. Using a compliant IdP is the preferable solution.
{{% /alert %}}
specific group, by performing the steps below. Using a compliant IdP is the
preferable solution.

1. User should ask to join the [GOCDB user

>    group](https://aai.egi.eu/registry/co_petitions/start/coef:41) ;

Reviewer: Not sure about how to handle punctuation in lists, we probably need a native speaker (@mviljoen-egi?) to help here.

Author: I am not using punctuation at end of list items, unless items contain more than one sentence (and then I add punctuation to all items). Please advise.

Reviewer: Please follow those rules: https://docs.openstack.org/doc-contrib-guide/writing-style/lists.html

Author: I am already compliant with list punctuations in most of the content that I wrote. As this is a numbered list, I added dots.

> To access the configuration database using a digital certificate, first obtain

Reviewer: ```suggestion
To access the Configuration Database using a digital certificate, first obtain
```

Author: I disagree with this. Using capitalized names everywhere in the text is a mistake. We should only do that when we are referring to the official service, and we need the full service name (maybe even with a link). Where we describe what a service actually is or when we discuss its features and capabilities, we should just use its name as a noun.

Reviewer: Here we are referring to the service name, hence it should be capitalised.

> (CAs), and install it on your local machine.

Reviewer: ```suggestion
(CAs), and install it on your internet browser.
```

Author: Well, I am functioning perfectly fine with the certificate in the cert store on my local machine, not added directly to one particular browser. So I will just include both of these options.

Reviewer: Yes, that's true that it depends on the OS and browser.

> {{% alert title="Note" color="info" %}} EGI Check-in does not support single

Reviewer: This is not EGI Check-in the problem here.

Author: Fixed

> {{% alert title="Note" color="info" %}} GOCDB does not support single

Reviewer: ```suggestion
{{% alert title="Note" color="info" %}} The Configuration Database does not support single
```
